src/patternfly/components/Drawer/examples/Drawer.md

```diff
@@ -5,6 +5,7 @@ cssPrefix: pf-c-drawer
 ---
 
 ## Examples
+
 ```hbs title=Closed
```

Comment:
Should this be "Closed panel on right", then "Expanded panel on right", then "Closed panel on left" and then "Expanded panel on left"

Reply:
Updated

Reply:
Updated